docs/js/tiles.js

```diff
@@ -28,11 +28,28 @@ function submitQuery() {
     .map(toKeyValueString)
     .join("&")
 }
+function tesselace(){
+
+  // keep tesselace reference as long as tile definition is unchanged
+  var tesselace = ""
+  if (window.location.search.substr(1).includes("tesselace=")) {
+    // obtain tile definition from url
+    var urlTile = window.location.search.replace(/(.*[?&])?tile=/, "").replace(/&.*/, "")
```

Comment:
I was thinking too quickly with this suggestion. There might be no query at all or no tile argument. That would not work.

Reply:
[URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) looks interesting, but IE does not support and is still [used](https://en.wikipedia.org/wiki/Usage_share_of_web_browsers#Summary_tables).

Reply:
If there is no query, then the check for "tesselace"+ will fail and there is no problem.  Similarly, you can't have "tesselace'+ without a tile unless someone has messed up the URL by hand.